app/views/shipping/index.scala.html

```diff
@@ -1,6 +1,8 @@
-@(shipping: model.ShippingDetails)
+@(shipping: model.ShippingDetails)(implicit rh: RequestHeader)
 
-@main(s"${shipping.name.split("\\s").map(_.capitalize).mkString(" ")} | Choose a Package | Subscriptions | The Guardian", footer = fragments.footer.intl_faq()) {
+@capitalizedName=@{shipping.name.split("\\s").map(_.capitalize).mkString(" ")}
```

Comment:
I think `@{shipping.name.split("\\s").map(_.capitalize).mkString(" ")}` should be in `ShippingDetails`